README.md

```diff
@@ -1,3 +1,25 @@
 # Spack Config
 
-Shared spack configuration for building software on gadi@NCI
+This repository contains:
+
+* Spack configuration files required by the Spack instance(s) maintained by ACCESS-NRI on Gadi and ACCESS-NRI's [CI Docker image](https://github.com/ACCESS-NRI/build-ci/).
+  * The `ci` directory is to be used by the CI Docker image.
+  * The `gadi` directory is to be used in Gadi deployments.
+  * The `common` directory is a subset of required configuration files and should not be used directly.
+
+* A dot source file, `spack-start.bash`, for developers to setup their `bash` environment to directly run `spack` commands using ACCESS-NRI's Spack instance(s). Execute: `. spack-start.bash`.
```

Comment:
```suggestion
* A dot source file, `spack-start.bash`, for maintainers to setup their `bash` environment to directly run `spack` commands using ACCESS-NRI's Spack instance(s). Execute: `. spack-start.bash`. Note: this is not for general users.
```

Reply:
This script in its current state is for general users. In the future, for advanced users, we may need to provide a customisable script or create a mechanism to read a configuration file from their home directory. Please see the updated README.

Reply:
Good point. The updated version is clearer, thanks.